consumer-test/integrationTestController.sh

```diff
@@ -57,6 +58,9 @@ TEST_CASES=$(find testCases -name '*.yml')
 while true; do sleep 10; echo "[INFO] Integration tests still running."; done &
 notificationThreadPid=$!
 
+while true; do sleep 40000; echo "[INFO] Here are the preliminary logs:"; ls -laR |grep -i 'log.txt' ;find . -name 'log.txt' |xargs cat; done &
```

Comment:
that is 40k seconds, so > 11 hours? that seems wrong.

Reply:
yes, that was terribly wrong. Should be 40 minutes since after ~50 Minutes the build gets killed by travis. It is 60x40 seconds now (2400).